packages/core/contracts/financial-templates/common/TokenFactory.sol

```diff
@@ -24,8 +24,6 @@ contract TokenFactory is Lockable {
         uint8 tokenDecimals
     ) external nonReentrant() returns (ExpandedIERC20 newToken) {
         SyntheticToken mintableToken = new SyntheticToken(tokenName, tokenSymbol, tokenDecimals);
-        mintableToken.addMinter(msg.sender);
```

Comment:
This change should be tested in [TokenFactory.js](https://github.com/UMAprotocol/protocol/blob/master/packages/core/test/financial-templates/TokenFactory.js), [EMPCreator.js](https://github.com/UMAprotocol/protocol/blob/master/packages/core/test/financial-templates/expiring-multiparty/ExpiringMultiPartyCreator.js) and the [PerpCreator.js](https://github.com/UMAprotocol/protocol/blob/master/packages/core/test/financial-templates/perpetual-multiparty/PerpetualCreator.js) unit test files.

Reply:
done